config/kube-vip-cloud-provider/overwrites/Chart.yaml

```diff
@@ -20,7 +20,8 @@ version: 0.2.2
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: v0.0.4
+# *** GIANT SWARM : Bump version to fix CVE. ***
+appVersion: v0.0.5
```

Comment:
By default the upstream chart references `v0.0.4` but there are fixes for CVEs in `v0.0.5` 👇
https://github.com/kube-vip/kube-vip-cloud-provider/releases/tag/v0.0.5
That chart doesn't seem to be maintained so much.
There are newer versions (0.0.9 but they don't seem groundbreaking).